integrations/tensorflow/python_projects/iree_tf/version_info.json

```diff
@@ -0,0 +1,3 @@
+{
+  "package-version": "3.0.0.dev"
```

Comment:
Updating `iree-tools-tf` and `iree-tools-tflite` could be out of scope for this current sprint of work. Maybe we can keep those on calver for now and push new packages sometime using the current versioning scheme? We could rename those wheels with the version change or use an epoch.

Reply:
Works and this is actually easily to revert if we don't name the new files `version_info.json`.